### PR TITLE
Add Hover Visual States to the Column Header.

### PR DESCRIPTION
**Pull request by mariaterzieva12**

### CODE REVIEW

File: controls/datagrid/columns/header.md [hunk 1]
```diff
@@ -66,6 +66,31 @@ Define the `HeaderContentTemplate` in the DataGridColumn:
...
 ![.NET MAUI DataGrid Column Header Template](../images/header-content-template.png)
 
+## Hover Visual State on Column Header
+
+You can change the background color of the column header on hover visual state by setting the `BackgroundColor` property.
+
+This example demonstrates how to apply `BackGroundColor` to the DataGrid Header on Hover Visual State: 
+
+```XAML
+ <VisualStateManager.VisualStateGroups>
```
Comment (didiyordanova) [inline controls/datagrid/columns/header.md:76]: Create sdk example for this case or include the approach in existing example inside the styling section. then add the code snippet to the documentation.

File: controls/datagrid/columns/header.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Column Headers
 page_title: .NET MAUI DataGrid Documentation - Columns Header
-description: Check our &quot;Columns Header&quot; documentation article for Telerik DataGrid for .NET MAUI.
+description: Review the Column Header article to learn how to style and customize the text in the Telerik UI for .NET MAUI DataGrid Header.
```
Comment (yordan-mitev) [inline controls/datagrid/columns/header.md:4]:
```suggestion
description: Learn how to style and customize the text in the Telerik UI for .NET MAUI DataGrid Header by using the properties of the component.
```

File: controls/datagrid/columns/header.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (yordan-mitev) [inline controls/datagrid/columns/header.md:1]: Please check the [first heading](https://github.com/telerik/maui-docs/blob/a24d52c04cadfe514bf36d5259864bd5ba3e5a80/controls/datagrid/columns/header.md#changing-the-text-in-the-header) in the article and edit them by implementing [title-case capitalization](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).

File: controls/datagrid/columns/header.md [hunk 4]
```diff
@@ -66,6 +66,31 @@ Define the `HeaderContentTemplate` in the DataGridColumn:
 
 ![.NET MAUI DataGrid Column Header Template](../images/header-content-template.png)
 
+## Hover Visual State on Column Header
```
Comment (yordan-mitev) [inline controls/datagrid/columns/header.md:69]:
```suggestion
## Color on Hover
```

File: controls/datagrid/columns/header.md [hunk 5]
```diff
@@ -66,6 +66,31 @@ Define the `HeaderContentTemplate` in the DataGridColumn:
...
+
+You can change the background color of the column header on hover visual state by setting the `BackgroundColor` property.
```
Comment (yordan-mitev) [inline controls/datagrid/columns/header.md:71]:
```suggestion
You can change the hover state background color of the column header by setting the `BackgroundColor` property.
```

File: controls/datagrid/columns/header.md [hunk 6]
```diff
@@ -66,6 +66,31 @@ Define the `HeaderContentTemplate` in the DataGridColumn:
...
+
+This example demonstrates how to apply `BackGroundColor` to the DataGrid Header on Hover Visual State:
```
Comment (yordan-mitev) [inline controls/datagrid/columns/header.md:73]:
```suggestion
The following example demonstrates how to apply the `BackGroundColor` property to the DataGrid header for its hover visual state: 
```

**Review by didiyordanova — APPROVED** [review]

**Pull request merged**